be/src/storage/tablet_meta.h

```diff
@@ -135,6 +135,7 @@ class TabletMeta {
     TabletTypePB tablet_type() const { return _tablet_type; }
     TabletUid tablet_uid() const;
     int64_t table_id() const;
+    void set_table_id(int64_t table_id);
```

Comment:
should mark as TEST only, regular code should never use this interface to change tablet id directly.

```suggestion
    void TEST_set_table_id(int64_t table_id);
```